intellij/org.eclipse.xtend.idea/src/org/eclipse/xtend/core/idea/config/XtendSupportConfigurable.xtend

```diff
@@ -71,7 +71,7 @@ class XtendSupportConfigurable extends FrameworkSupportInModuleConfigurable {
 		val parent = sibling?.file?.parent
 		if (parent === null)
 			return null
-		val existing = parent.children.findFirst[it.name == name]
```

Comment:
can we use com.intellij.openapi.vfs.VfsUtil.createDirectoryIfMissing(VirtualFile, String) ?